include/deal.II/distributed/tria_base.h

```diff
@@ -188,6 +191,13 @@ namespace parallel
      */
     types::subdomain_id n_subdomains;
 
+    /**
+     * Flag indicating if the vector n_locally_owned_active_cells has been
+     * filled. It is only filled need based since in the regular simulations
+     * this information is not needed.
+     */
+    bool n_locally_owned_active_cells_per_processor_has_been_updated;
```

Comment:
Do you really need this variable? One should be able to query it from `n_locally_owned_active_cells.size()`

Reply:
That is a good idea!